lib/ansible/modules/windows/win_say.ps1

```diff
@@ -36,7 +36,7 @@ $result = @{
 $speed = 0
 $words = $null
 
-if ($speech_speed -ne $null) {
+if ($speech_speed) {
```

Comment:
My reasoning was that if you put the speed to zero, it is the same as not setting the speed at all (as the range is between -10 to 10, from slow to fast).

Zero means normal speed.